survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
+  EC2_INSTANCE_TAG = "RabbitMQ"
+  RETENTION_DAYS   = "${var.ebs_snapshot_retention_days}"
+  unique_name      = "V1"
```

Comment:
Can we make this "rabbitmq_ebs_snapshot"?
Currently the lambda function is called "jonshaw_lambda_V1" which is very nondescript.
If we change this it will become "jonshaw_lambda_rabbitmq_ebs_snapshot"